Sources/SourceKitLSP/Swift/SemanticRefactoring.swift

```diff
@@ -103,41 +60,41 @@ extension SemanticRefactoringError: CustomStringConvertible {
 }
 
 extension SwiftLanguageService {
-  /// Provides detailed information about the result of a specific refactoring operation.
+
+  /// Handles the `SemanticRefactorCommand`.
   ///
-  /// Wraps the information returned by sourcekitd's `semantic_refactoring` request, such as the necessary edits and placeholder locations.
+  /// Makes request to sourcekitd and wraps the result into a
+  /// `SemanticRefactoring` and then makes an `ApplyEditRequest` to the client
+  /// side for the actual refactoring.
   ///
   /// - Parameters:
-  ///   - refactorCommand: The semantic refactor `Command` that triggered this request.
+  ///   - semanticRefactorCommand: The `SemanticRefactorCommand` that triggered this request.
+  ///
+  /// - Returns: A `WorkspaceEdit` with the necessary refactors as a `LSPAny`
```

Comment:
I think this comment is out-of-date, right? This function doesn’t return a `WorkspaceEdit`. It actually got me wondering: Do we need to return anything here at all? Similar to my comment on `macroExpansion`, I wouldn’t expect the client to interpret the `LSPAny` and we already handled the edits by sending `ApplyEditRequest` to the client.

Reply:
Here, I would like to highlight the reply which I made to your comment on `macroExpansion`

Reply:
Just to double-check: You mean [this comment](https://github.com/apple/sourcekit-lsp/pull/1436#discussion_r1647371565), right? Saying that we should change this to not return the edits, at which point the comment will go away, right?